src/azure-cli/azure/cli/command_modules/dms/_help.py

```diff
@@ -51,7 +51,7 @@
 examples:
   - name: Create an instance of DMS.
     text: >
-        az dms create -l westus -n mydms -g myresourcegroup --sku-name Basic_2vCores --subnet /subscriptions/{vnetSubscriptionId}/resourceGroups/{vnetResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{subnetName} --tags tagName1=tagValue1 tagWithNoValue
```

Comment:
Should we mention above that this parameter is available, although it is optional and it defaults to the resource group location? Since resource groups have global scope, someone may wish to create "MySQL migrations" resource group and add a few DMS services in multiple regions, for example. And it is not clear how this could be done from the help.